docs/pages/dsFPD/ds-front-panel-display_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,185 @@
+
+## Level 3 Python Test Cases High Level Overview
+
+The class diagram below illustrates the flow of dsFPD L3 Python test cases:
+
+```mermaid
+---
+title: dsFPD - Python Class Flow
+---
+classDiagram
+    testControl <|-- ut_raft
+    class ut_raft{
+    }
+    ut_raft <|-- dsFPD
+    dsFPD <|-- L3TestClasses
+```
+
+- **testControl**
+  - Test Control Module for running rack Testing. This module configures the `DUT` based on the rack configuration file provided to the test.
+  - This class is defined in `RAFT` framework. For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/1.0.0/README.md)
+- **ut_raft**
+  - Python based testing framework for writing engineering tests.
+  - It provides common functionalities like menu navigation, configuration reader, reading user response etc.
+  - For more details [ut-raft](https://github.com/rdkcentral/ut-raft).
+- **dsFPD**
+  - This is test helper class which communicates with the `L3` C/C++ test running on the `DUT` through menu
+- **L3TestClasses**
+  - These are the L3 test case classes
+  - Each class covers the each test use-case defined in [L3 Test use-cases](#level-3-test-cases-high-level-overview) table
+
+## YAML File Inputs
+
+```mermaid
+---
+title: dsFPD - YAML
+---
+classDiagram
+    class Config {
+        - rackConfiguration
+        - platform
+    }
+    class DeviceConfig {
+        - cpePlatformConfiguration
+    }
+    class PlatformProfile {
+        - componentConfigurations
+    }
+    PlatformProfile <|-- DeviceConfig
+    DeviceConfig <|-- Config
+    class TestSetupConfig {
+        - testSetupConfiguration
+    }
+    class MenuConfig {
+        - menuConfiguration
+    }
+    MenuConfig <|-- TestSetupConfig
+```
+
+- **config.yaml**
+  - Identifies the rack configuration and platform used
+  - References platform-specific config from `deviceConfig.yaml`
+  - For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/1.0.0/README.md) and [example_rack_config.yml](https://github.com/rdkcentral/python_raft/blob/1.0.0/examples/configs/example_rack_config.yml)
+
+- **deviceConfig.yaml**
+  - Specifies overall configuration for the platform
+  - Can be overridden by:
+    - Changing locally .yaml file directory
+    - Using --deviceConfig command line switch
+  - For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/1.0.0/README.md) and [example_device_config.yml](https://github.com/rdkcentral/python_raft/blob/1.0.0/examples/configs/example_device_config.yml)
+
+- **componentProfile.yaml/platformProfile.yaml**
+  - Contains component-specific configurations
+  - Contains platform wide configuration broken down into separate components
+  - Example configuration file [dsVideoPort_Settings](https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/main/profiles/sink/Sink_4K_VideoPort.yaml)
```

Comment:
please provide the front panel configuration file

Reply:
changed the link to FPD configuration file